src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -123,7 +121,20 @@ protected override int ExecuteCmd(DocumentsOperationContext context)
                             case CounterOperationType.Increment:
                             case CounterOperationType.Delete:
                             case CounterOperationType.Put:
-                                LoadDocument();
+                                try
+                                {
+                                    LoadDocument();
+                                }
+                                catch (DocumentDoesNotExistException)
+                                {
+                                    if (_fromSmuggler)
+                                    {
+                                        ErrorCount++;
```

Comment:
Are we doing something with that? We need to increase the error count in the progress, right?